packs/tfe_fdo_nomad/README.md

```diff
@@ -0,0 +1,156 @@
+# Nomad pack for Terraform Enterprise FDO
+
+<!-- Include a brief description of your pack -->
+
+This pack deploys Terraform Enterprise on Nomad. This includes running a Terraform Enterprise service job and Terraform Enterprise agent batch job.
+
+## Pack Usage
+
+The pack expects certain prerequisites to be fulfilled before running. The list of prerequisites are: 
+
+Interacting with the Nomad server will require
+1. `NOMAD_ADDR` - The address of the Nomad server.
+1. `NOMAD_TOKEN` - The SecretID of an ACL token to use to authenticate API requests with.
+1. `NOMAD_CACERT` - Path to a PEM encoded CA cert file to use to verify the Nomad server SSL certificate.
+1. `NOMAD_CLIENT_CERT` - Path to a PEM encoded client certificate for TLS authentication to the Nomad server. Must also specify NOMAD_CLIENT_KEY.
+1. `NOMAD_CLIENT_KEY` - Path to an unencrypted PEM encoded private key matching the client certificate from NOMAD_CLIENT_CERT.
+
+After setting up the environment variables, the pack can be setup using the following steps:
+
+1. Create Namespace for TFE job and TFE agent job.
+
+   1. Run `nomad namespace apply terraform-enterprise` to create the `terraform-enterprise` namespace. This is the default namespace that is used to bring up TFE Job.
+   1. Run `nomad namespace apply tfe-agents` to create the `tfe-agents` namespace. This is the default namespace that is used to bring up TFE Agent Job.
+
+
+2. Create a Nomad ACL policy file `terraform_enterprise_policy.hcl` with the content below:
+```hcl
+  namespace "tfe-agents" {
+  capabilities = ["submit-job","dispatch-job", "list-jobs", "read-job", "read-logs" ]
+  }
+  ```
+
+3. Apply the Nomad policy using:
+  ```bash
+  $ nomad acl policy apply \
+   -namespace terraform-enterprise -job tfe-job \
+   -group tfe-group -task tfe-task \
+   terraform-enterprise-policy ./terraform_enterprise_policy.hcl
+  ``` 
+
+4. Create the necessary Nomad Variables for each job. 
+  
+  These contain sensitive data that are required like certs, licenses and passwords. 
+  Create a variable specification file: 
+
+  ```hcl
+  # spec.nv.hcl
+  path      = "nomad/jobs/tfe-job"
```

Comment:
It should be mentioned that this needs to be adapted if the `job_name` Pack variable has changed